x-pack/metricbeat/module/gcp/billing/billing_test.go

```diff
@@ -5,11 +5,16 @@
 package billing
 
 import (
-	"io/ioutil"
+	"io"
 	"log"
 	"strconv"
 	"testing"
 
+	"cloud.google.com/go/bigquery"
```

Comment:
bigquery was used before as well. How was it working without inclusion of this library ?

Reply:
It's because the function that translates the tags string into a map expects an argument of type `bigquery.Value`.

https://github.com/elastic/beats/blob/61e984ac49abb059fdd0198de38e979b8ab4e2eb/x-pack/metricbeat/module/gcp/billing/billing_test.go#L46